robot/basestation/stream_capture.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
```

Comment:
All the newly added functions should have [docstrings](https://www.datacamp.com/community/tutorials/docstrings-python)